composer.json

```diff
@@ -25,7 +25,8 @@
 	"require": {
 		"silverstripe/admin": "^1",
 		"silverstripe/framework": "^4",
-		"ua-parser/uap-php": "^3.5"
+		"ua-parser/uap-php": "^3.5",
+        "silverstripe/crontask": "^2.0"
```

Comment:
Would this make more sense to be in `suggested`, rather than being a hard requirement, since it's not the only option for getting garbage collection working?

Reply:
(Might be a good time to reformat the indentation in this file, too.)

Reply:
That wouldn't make sense as we use the cron module for the cron task.
I also think that splitting the task out to another module is overkill.

Reply:
You can include integrations in a module without mandating the presence of the other module - see [Elemental's FullTextSearch integration](https://github.com/dnadesign/silverstripe-elemental/blob/4/src/Search/ElementalSolrIndex.php) for an example.

I completely agree that a separately module would be overkill, so I wouldn't recommend that approach.

Reply:
I'm not sure how that would work as `\SilverStripe\CronTask\Controllers\CronTaskController::index` uses `ClassInfo::implementorsOf(CronTask::class)` which seems to ignore the return statement as it's parsing the file using `PhpParser` in the `ClassManifest`.

Reply:
Fixed (needed to use `interface_exists` instead of `class_exists`.)